### ISSUE TITLE
Question about the parameter of upsampling in FCN32s

### ISSUE THREAD

**Issue by super233** [ZijunDeng/pytorch-semantic-segmentation#56]
&emsp;&emsp;According to the formula of ConvTranspose2d: **output = (input - 1) * stride - 2 * padding + kernel_size**. To realize 32 times upsampling, the parameter should be **(kernel_size=32, stride=32, padding=0)**.
&emsp;&emsp;But the parameter in code is `self.upscore = nn.ConvTranspose2d(num_classes, num_classes, kernel_size=64, stride=32, bias=False)`.
&emsp;&emsp;Could you please tell me why it's this? Thank you : )
